src/tyr/route_serializer_osrm.cc

```diff
@@ -1378,12 +1296,16 @@ std::string serialize(valhalla::Api& api) {
     // Create a route to add to the array
     auto route = json::map({});
 
-    // TODO: phase 1, just hardcode score. phase 2: do real implementation
-    if (options.action() == valhalla::Options::trace_route)
+    if (options.action() == valhalla::Options::trace_route) {
```

Comment:
this should not be limited to map matching.

Reply:
Yep